WooCommerce/Classes/ViewRelated/Dashboard/Settings/Beta features/BetaFeaturesViewController.swift

```diff
@@ -144,8 +145,8 @@ private extension BetaFeaturesViewController {
     func configureProductsDescription(cell: BasicTableViewCell) {
         configureCommonStylesForDescriptionCell(cell)
 
-        let description = NSLocalizedString("Test out new product editing functionalities as we get ready to launch them",
-                                            comment: "My Store > Settings > Experimental features > Product editing")
+        let description = NSLocalizedString("Test out the new products section as we get ready to launch them",
```

Comment:
If we will update the title, probably we should update also the description.

Reply:
(responded in https://github.com/woocommerce/woocommerce-ios/pull/3008#issuecomment-712530545)